api/resolvers/wallet.js

```diff
@@ -732,8 +738,8 @@ export const walletLogger = ({ wallet, models }) => {
   const log = (level) => async (message, context = {}) => {
     try {
       if (context?.bolt11) {
-        // automatically populate context from bolt11 to avoid duplicating this code
-        const decoded = await parsePaymentRequest({ request: context.bolt11 })
+        // automatically populate context from invoice to avoid duplicating this code
+        const decoded = await parseInvoice({ request: context.bolt11, lnd })
```

Comment:
Mhh, I think ideally this would store the bolt12 offer as `bolt12` in the context instead of the bolt12 invoice as `bolt11`.

But I am not sure how easy it is to get the offer back from the invoice? But I think we can at least save it as `bolt12` here.

_I really have to do #1598 soon :eyes:_

Reply:
i don't think the offer is available as part of the [invoice tlv](https://github.com/lightning/bolts/blob/master/12-offer-encoding.md#tlv-fields-for-invoice_request). 
Can we do all the cosmetic changes from bolt11->bolt12 on a separated pr? This one is already big, propagating all the naming would add even more changes. Unless you mean something else

Reply:
> Can we do all the cosmetic changes from bolt11->bolt12 on a separated pr? This one is already big, propagating all the naming would add even more changes. Unless you mean something else

I just meant the user-facing stuff (so not just cosmetic imo), not renaming variables but yeah, we can do it in a separate PR. Just wanted to flag this.